src/components/backend-ai-storage-list.ts

```diff
@@ -1648,15 +1669,18 @@ export default class BackendAiStorageList extends BackendAIPage {
       return;
     }
     if (typeof globalThis.backendaiclient === 'undefined' || globalThis.backendaiclient === null || globalThis.backendaiclient.ready === false) {
-      document.addEventListener('backend-ai-connected', () => {
+      document.addEventListener('backend-ai-connected', async () => {
         this.is_admin = globalThis.backendaiclient.is_admin;
         this.enableStorageProxy = globalThis.backendaiclient.supports('storage-proxy');
         this.enableInferenceWorkload = globalThis.backendaiclient.supports('inference-workload');
         this.authenticated = true;
         this._APIMajorVersion = globalThis.backendaiclient.APIMajorVersion;
         this._maxFileUploadSize = globalThis.backendaiclient._config.maxFileUploadSize;
-        this._checkFilebrowserSupported();
-        this._refreshFolderList(false, 'viewStatechanged');
+        await this._checkFilebrowserSupported();
+        await this._getVolumeInformation();
+        await this._getAllowedVFolderHostsByCurrentUserInfo();
+        await this._triggerFolderListChanged();
+        await this._refreshFolderList(false, 'viewStatechanged');
```

Comment:
Is there a reason to `await` every method? Due to these lines, the folder list items are displayed after some white screen. Especially, when I clicked the "AUTOMOUNT FOLDERS", it displayed no items for 2-3 seconds even though I have some dot folders.

Reply:
I removed the `await` keywords here after some testing. Please let me know if it has a side-effect.